### ISSUE TITLE
@package tag is not used anymore in 7.0.0-alpha

### ISSUE THREAD

**Issue by clemsciences** [ApiGen/ApiGen#1081]
I am trying to use the new version of ApiGen and I see that packages are not used anymore to structure the docs and only the namespaces do that. 
Is it possible to retrieve the package structure?

**Comment by JanTvrdik** [comment]
`@package` tag should be parsed as any other tag and available e.g. in `$classLike->tags['package']`, but ApiGen itself does neither use it nor index it.

**Comment by clemsciences** [comment]
Ok, so you mean that it is a package used by ApiGen that has changed its way to handle the `@package` tag?

---------------------------

It was like this before:

![Screenshot_1](https://user-images.githubusercontent.com/13933660/190643767-415d366d-0533-4515-8a4f-41b0321b7a7e.png)
--------------------------------

But now, it is like this:
![Screenshot_2](https://user-images.githubusercontent.com/13933660/190644342-94c2164c-3b7b-4d4b-90e0-2c112de486ff.png)

--------------------------

![Screenshot_3](https://user-images.githubusercontent.com/13933660/190644357-f60a336d-2c1c-48da-962c-e6625f1fbe52.png)

--------------------------------

The docs had a **packages** section and now they don't have it anymore. And I didn't change my source code.

Code to generate the docs:
```bash
docker run --rm --volume "$PWD:$PWD" --workdir "$PWD" apigen/apigen server/api --output docs --title="My source"
```
The source of the docs is found in `server/api` directory and the output directory is `docs`. What did I miss if you said that nothing changed?

**Comment by JanTvrdik** [comment]
Let me rephrase this. The `@package` is no longer supported. You should organize your code with namespaces.

>  Is it possible to retrieve the package structure?

You can build the package index yourself but ApiGen does not provide it anymore.

~~~php
$packageIndex = [];
foreach ($index->classLike as $info) {
  foreach ($info->tags['package'] ?? [] as $value) {
    $packageIndex[$value->value][] = $info;
  }
}
~~~

**Comment by clemsciences** [comment]
Ok, I didn't understand your first reply. So this is a major change and it was not mentioned anywhere. I cannot unfortunately change my code to use namespaces.

What is the code you show supposed to do? And where is it used (or should be used)?

**Comment by clemsciences** [comment]
Is it possible to re-add this feature in a future release? I could try to contribute to this code but I would like to be sure that it would be accepted.

**Comment by JanTvrdik** [comment]
Probably not, or at least not anytime soon.

**Comment by clemsciences** [comment]
Ok, I'm trying to do it myself and will do a pull request if it works.

**Comment by clemsciences** [comment]
Well, I tried my best, I coded templates and the parsers, but I can't make it work. If someone is interested in helping me, it would be a pleasure to collaborate: https://github.com/clemsciences/ApiGen.
